src/Security/KbinAuthenticator.php

```diff
@@ -56,7 +56,7 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token,
         /** @var User $user */
         $user = $token->getUser();
 
-        return new RedirectResponse($this->urlGenerator->generate($user->homepage));
```

Comment:
this is an interesting change; I think it's unrelated to the UI filter changes? Looking at it, I assume it works because the front controller itself redirects based on user. it looks like `$user` is unused now, so maybe no longer needs to be fetched? Unless there's some magic PHP thing it's doing

Reply:
Oh, you're right - I just wasn't looking; I don't think we don't need to fetch $user anymore in that codepath no. I'll push the appropriate fix.

And it's... tangentially related to the UI filter code. In order to do the filtering, I had to deduplicate some of the code between what used to be different routes for the home page, and make it all one route. So, now that we only have one route, this change (a) keeps things working with the new code and (b) is a good cleanup anyway, since IMO this code shouldn't have to care about user settings or routes.

Reply:
My only slight concern if this code was there for a reason; I suppose it makes sure the user gets their desired homepage on login, otherwise it relies on the symfony redirect handler. Locally, I didn't see an issue, but local dev env doesn't use nginx or other rewrite handlers. I _think_ @nobodyatroot saw an issue with the redirect component not working, but I can't recall if that was an nginx thing, or cloudflare thing

Reply:
Yeah, definitely - I've replaced the functionality that this was implementing (making sure that people got their desired homepage) with a different implementation, not just removed it.

We used to make that decision by sending people down one of a variety of routes, but now 'front' does all that, so it should be safe to just send everyone to 'front'.